app/src/main/java/com/epfl/drawyourpath/userProfile/dailygoal/DailyGoal.kt

```diff
@@ -0,0 +1,58 @@
+package com.epfl.drawyourpath.userProfile.dailygoal
+
+import java.time.LocalDate
+
+/**
+ * class representing the Daily Goal that the user set
+ */
+data class DailyGoal(
+    var distanceInKilometerGoal: Double,
+    var timeInMinutesGoal: Double,
+    var nbOfPathsGoal: Int,

```

Comment:
maybe you can rename the time in activityTime to be in accordance with the rest of the code